pkg/config/config.go

```diff
@@ -82,7 +82,8 @@ type etcd struct {
 }
 
 type s3 struct {
-	Bucket string `env:"ZINC_S3_BUCKET"`
+	Bucket   string `env:"ZINC_S3_BUCKET"`
+	Endpoint string `env:"ZINC_S3_ENDPOINT"`
```

Comment:
Great addition but maybe we should keep the naming inline with the aws sdk so instead of using ZINC_S3_ENDPOINT we could define the property as ZINC_S3_URL ?